packages/create-cli/src/actions/template.ts

```diff
@@ -22,7 +22,7 @@ export async function copyTemplate ({ template, templatePath, targetDir }: CopyT
       downloadTemplateSpinner.text = chalk.red(`Couldn't find template "${template}"`)
       downloadTemplateSpinner.fail()
     } else {
-      console.error(e.message)
+      process.stderr.write(e.message)
```

Comment:
There is a difference between console and stderr.write. console operations always add a new line whereas stderr.write doesn't